app/models/study.rb

```diff
@@ -819,13 +819,27 @@ def can_edit?(user)
     end
   end
 
+  # google allows arbitrary periods in email addresses so if the email is a gmail account remove any excess periods
+  def google_email_period_remover(email_address)
+    email_address = email_address.downcase
+    if email_address.end_with?('gmail.com')
+      # sub out any periods with blanks, then replace the period for the '.com' at the end of the address
+      email_address = email_address.gsub('.','').gsub(/com\z/,'.com')
```

Comment:
Non-blocking: this function throws a few additional lint issues in my VS Code, but the lack of space after comma is the only one that slightly degrades readability for me.

```suggestion
      email_address = email_address.gsub('.', '').gsub(/com\z/, '.com')
```

(I added a [how-to: linter set up for Ruby in VS Code](https://docs.google.com/document/d/1lRYQB3zQB2iHpdE8LFu88a-b74qnIm0VUZTMFv4eOXM/edit#bookmark=id.qsrwtn9sm0pg) to our onboarding docs a couple months ago, FWIW.)